build.gradle

```diff
@@ -0,0 +1,43 @@
+plugins {
```

Comment:
ทำไมมี ทั้ง gradle และ sbt หล่ะครับ

Reply:
ตอนที่สร้าง project ด้วย sbt มันก็มีทั้งสองตัวมาเลยอ่ะครับ ตอนนี้สับสนแล้วครับว่ามันใช้ตัวไหน build กันแน่

Reply:
แปลว่าสร้างผิด ปกติสร้างยังไงก็ไม่มี gradle ติดมา

Reply:
ผมสร้างตาม doc เลยครับ run คำสั่งนี้เลย

`sbt new playframework/play-scala-seed.g8`

https://www.playframework.com/documentation/2.6.x/NewApplication

Reply:
ใน example ของเขาก็มีอ่ะครับ ตกลงมันใช้ตัวไหนกันแน่ครับเนี่ย
https://github.com/playframework/play-scala-starter-example/tree/2.6.x

Reply:
ไม version ใหม่มีหว่าาา เก่าๆไม่เคยมี งั้นก็เลือกเอา จะใช้ gradle หรือใช้ sbt อีกตัวกะลบไปเหอะ...

Reply:
ลองลบ gradle แล้วมันโหลดให้ใหม่เลยครับ พวก gradle-wrapper, gradlew ยกเว้น build.gradle

Reply:
ลบ build.gradle ไปด้วยครับ

Reply:
ลบ build.gradle ได้ครับ แต่ folder gradle กับ gradlew ลบแล้วมันจะโหลดใหม่ครับ หรือจะให้ผมลบ build.gradle อย่างเดียว

Reply:
ผมลองไปหาข้อมูลมา เห็นมีหลายคนที่ใช้ play ใน version ก่อนๆหน้านี้ ถามใน stackoverflow ถึงการเอา gradle มา build ใน play คู่กับ sbt ซึ่งผมไม่รู้ว่าเอามาใช้ build อะไรเหมือนกัน และก็มีบางบทความพูดถึง gradle ใน play เมื่อหลายปีก่อน

https://medium.com/@reactivedata/gradle-play-framework-part-i-7375305ad9a3
https://engineering.linkedin.com/play/developing-play-applications-using-gradle

*คหสต. บางที version นี้ dev เขาอาจจะใจดีใส่มาให้ก็ได้ครับ
*คำถามนี้เคยมีคนถามใน stackoverflow ด้วยแต่บินไปแล้ว
https://stackoverflow.com/questions/48780287/play-2-6-scala-why-gradle-is-added-to-the-starter-application-with-sbt

Reply:
พี่ลอง
$> rm -Rf gradle*
(แปลว่าไม่มีอะไรเกี่ยวกะ gradle เหลือเลย)
แล้วก็ run ได้นะ

Reply:
งั้นผมใส่ ignore gradle ของโปรเจคไว้นะครับ แต่ในเครื่องอาจจะมี gradle อยู่ เพราะพอลบไป intellij มันจะโหลดกลับมาอ่ะครับ

Reply:
ได้ครับ